plugins/woocommerce-admin/client/customize-store/assembler-hub/layout.tsx

```diff
@@ -57,106 +66,115 @@ export const Layout = () => {
 	const [ gradientValue ] = useGlobalStyle( 'color.gradient' );
 
 	return (
-		<div className={ classnames( 'edit-site-layout' ) }>
-			<motion.div
-				className="edit-site-layout__header-container"
-				animate={ 'view' }
-			>
-				<SiteHub
-					as={ motion.div }
-					variants={ {
-						view: { x: 0 },
-					} }
-					isTransparent={ isResizableFrameOversized }
-					className="edit-site-layout__hub"
-				/>
-			</motion.div>
-
-			<div className="edit-site-layout__content">
-				<NavigableRegion
-					ariaLabel={ __( 'Navigation', 'woocommerce' ) }
-					className="edit-site-layout__sidebar-region"
+		<LogoBlockContext.Provider
```

Comment:
Only added `<LogoBlock Context.Provider` here. The other changes are just indent.